code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -131,6 +132,11 @@
 	icon_state = "cart-b"
 	access_flora = 1
 */
+/obj/item/weapon/cartridge/robotics
+	name = "R.O.B.U.T.T. Cartridge"
```

Comment:
Needs `\improper` here.

Reply:
I don't see why it needs one, none of the other ones use it.

Reply:
Speaking of, why do people keep shoving improper down my throat? I keep things uniform with the others that no one has made issues about ever but my changes suddenly need them?

Reply:
issue resolved in discord